notNeededPackages.json

```diff
@@ -3785,6 +3785,10 @@
             "libraryName": "markdown-it-anchor",
             "asOfVersion": "7.0.0"
         },
+        "markdown-it-highlightjs": {
+            "libraryName": "markdown-it-highlightjs",
+            "asOfVersion": "4.0.0"
```

Comment:
I'd probably do `4.0.1`, since `4.0.0` had forgotten to include types in the package.

Reply:
Done.